src/nwm_routing/src/nwm_routing/__main__.py

```diff
@@ -1207,11 +1218,6 @@ def main_v03(argv):
                 forcing_end_time = time.time()
                 task_times['forcing_time'] += forcing_end_time - route_end_time
 
-            q0 = new_nwm_q0(run_results)
-
-            # TODO: Confirm this works with Waterbodies turned off
-            waterbodies_df = get_waterbody_water_elevation(waterbodies_df, q0)
```

Comment:
This content has been moved out of the `if run_set_iterator < len(run_sets) - 1:` condition